phpunit.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/Bootstrap.php"
+         colors="true"
+         testSuiteLoaderClass="PHPUnit_Runner_StandardTestSuiteLoader"
+         backupGlobals="false"
+         backupStaticAttributes="false">
```

Comment:
`backupStaticAttributes` is false by default, so this one could also be removed.